app/js/components/fb-feed.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component, Containers } from 'react';
+
+import Story from './fb-story';
+
+export default class Feed extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {feed:[]}
+  }
+
+  componentDidMount() {
+    var that = this;
```

Comment:
you may be able to get away with using all arrow functions, but this works as well